jwst/lib/tests/test_set_telescope_pointing.py

```diff
@@ -144,7 +144,7 @@ def test_pointing_averaging(eng_db_jw703):
         3.41404261e-06,  9.99994300e-01, -3.38429548e-03
     ])
     fsmcorr_exp = np.array([-0.02558673, -0.00200601])
-    obstime_exp = Time(1559582740.4880004, format='unix')
+    obstime_exp = Time(1559582740.488000, format='unix')
```

Comment:
If this is resilient, that is great. However, it may be that the check at 160 should use np.isclose.

Reply:
That makes more sense.  Changing.